### ISSUE TITLE
Ability for stack services to access secrets

### ISSUE THREAD

**Issue by ianwalter** [okteto/okteto#1016]
**Is your feature request related to a problem? Please describe.**

Stack services should be able to be configured to have access to secrets as environment variables. My use case is simple: I want to set `POSTGRES_PASSWORD` for PostgreSQL and have both the service container and my app be able to use it without hardcoding it in my stack file.

**Describe the solution you'd like**

If I add a secret called `DB_PASS` I should be able to access it in the stack file like:

```yaml
environment:
  - POSTGRES_PASSWORD=$DB_PASS
```

**Describe alternatives you've considered**

There is another issue requesting support for a `.env` file but to me this seems even more of a basic feature than that does. The solution I am requesting would be more straightforward but either solution would unblock me.

**Additional context**

Service logs before container fails:

db-594bf6ff74-68cf4dbError: Database is uninitialized and superuser password is not specified.
db-594bf6ff74-68cf4db You must specify POSTGRES_PASSWORD to a non-empty value for the
db-594bf6ff74-68cf4db superuser. For example, "-e POSTGRES_PASSWORD=password" on "docker run".
db-594bf6ff74-68cf4db
db-594bf6ff74-68cf4db You may also use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all
db-594bf6ff74-68cf4db connections without a password. This is *not* recommended.
db-594bf6ff74-68cf4db
db-594bf6ff74-68cf4db See PostgreSQL documentation about "trust":
db-594bf6ff74-68cf4db https://www.postgresql.org/docs/current/auth-trust.html

**Comment by pchico83** [comment]
@ianwalter thanks for the feedback. There is already an issue that I think is suggesting the same feature here:
https://github.com/okteto/cloud-feedback/issues/47
Should we close this one?

**Comment by ianwalter** [comment]
Yes, sorry I didn't see this issue when I searched.
